src/main/kotlin/minesweeper/domain/Cell.kt

```diff
@@ -4,4 +4,13 @@ sealed class Cell
 
 class Mine : Cell()
 
-class Land : Cell()
+class Land : Cell() {
+    private var _adjacentMines = 0
+
+    val adjacentMines: Int
+        get() = _adjacentMines
```

Comment:
```suggestion
    var adjacentMines: Int
        private set
```
컬렉션타입이 아닌 경우 위와 같이 외부에서의 변경을 막을수 있을것 같습니다 !